profiling/src/exporter/mod.rs

```diff
@@ -146,15 +146,16 @@ impl ProfileExporter {
             tags,
         })
     }
-
     /// Build a Request object representing the profile information provided.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
this is just for now (PR-time) to get your insight on this:
should I group `start` and `end` into a struct to it will reduce the number of parameters?
I could also group `additional_tags`, `endpoint_counts` and `metrics` into a struct `additional_info` ?